virtualbox/vbox-adapter-check.py

```diff
@@ -3,56 +3,83 @@
 import sys
 import textwrap
 import argparse
-import virtualbox
-from virtualbox.library import NetworkAttachmentType as NetType
+import subprocess
+import re
+import time
+import gi
+gi.require_version('Notify', '0.7')
 from gi.repository import Notify
+from vboxcommon import *
```

Comment:
recommend using isort for formatting consistency

Reply:
and black for other formatting consistency

Reply:
Good idea @williballenthin! We can address this after this has been merged in a different PR, added to https://github.com/mandiant/flare-vm/issues/507#issuecomment-2538569777